app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -239,8 +238,6 @@ val networkModule = module {
             EventSubTopic::class.java, Feedback::class.java, Speaker::class.java,
             EventFAQ::class.java)
 
-        onlineApiResourceConverter.enableSerializationOption(SerializationFeature.INCLUDE_RELATIONSHIP_ATTRIBUTES)
```

Comment:
Why removed?

Reply:
{
"data": {
"type": "feedback",
"attributes": {
"rating": "2.5",
"comment": "Awesome"
},
"relationships": {
"event": {
"data": {
"type": "event",
"id": "770"
}
},
"user": {
"data": {
"type": "user",
"id": "7625"
}
}
}
},
"included": [
{
"type": "user",
"id": "7625",
"attributes": {}
},
{
"type": "event",
"id": "770",
"attributes": {}
}
]
} 
becuase of this

Reply:
Explain please

Reply:
OK. Got it. But won't it effect past modules

Reply:
nope I have checked it isn't affecting any modules it may have broken some code before but everything is working fine